.eslintrc

```diff
@@ -26,35 +31,35 @@
         "eqeqeq": [ 2, "allow-null" ],
         "eol-last": 1,
         "indent": [ 1, "tab", { "SwitchCase": 1 } ],
-        'jsx-a11y/accessible-emoji': 2,
-        // 'jsx-a11y/anchor-has-content' doesn't handle our translation system
-        'jsx-a11y/aria-activedescendant-has-tabindex': 2,
-        'jsx-a11y/aria-props': 2,
-        'jsx-a11y/aria-proptypes': 2,
-        'jsx-a11y/aria-role': 2,
-        'jsx-a11y/aria-unsupported-elements': 2,
-        'jsx-a11y/click-events-have-key-events': 2,
-        'jsx-a11y/heading-has-content': 2,
-        'jsx-a11y/href-no-hash': 2,
-        'jsx-a11y/html-has-lang': 2,
-        'jsx-a11y/iframe-has-title': 2,
-        'jsx-a11y/img-has-alt': 2,
-        'jsx-a11y/img-redundant-alt': 2,
-        // 'jsx-a11y/label-has-for' Disabled because it doesn't check nested labels, which are valid
-        'jsx-a11y/lang': 2,
-        'jsx-a11y/mouse-events-have-key-events': 2,
-        'jsx-a11y/no-access-key': 2,
-        // 'jsx-a11y/no-autofocus' Replace this with a rule that detects if all autofocus have a describedby
-        'jsx-a11y/no-distracting-elements': 2,
-        'jsx-a11y/no-onchange': 2,
-        'jsx-a11y/no-redundant-roles': 2,
-        // 'jsx-a11y/no-static-element-interactions' Not needed right now if the following two rules are set
-        'jsx-a11y/onclick-has-focus': 2,
-        'jsx-a11y/onclick-has-role': 2,
-        'jsx-a11y/role-has-required-aria-props': 2,
-        'jsx-a11y/role-supports-aria-props': 2,
-        'jsx-a11y/scope': 2,
-        'jsx-a11y/tabindex-no-positive': 2,
+        "jsx-a11y/accessible-emoji": 2,
+        // "jsx-a11y/anchor-has-content" doesn"t handle our translation system
+        "jsx-a11y/aria-activedescendant-has-tabindex": 2,
+        "jsx-a11y/aria-props": 2,
+        "jsx-a11y/aria-proptypes": 2,
+        "jsx-a11y/aria-role": 2,
+        "jsx-a11y/aria-unsupported-elements": 2,
+        "jsx-a11y/click-events-have-key-events": 2,
+        "jsx-a11y/heading-has-content": 2,
+        // "jsx-a11y/href-no-hash": 2,
```

Comment:
I know these rules have changed but we shouldn't leave the deprecated rules here. What if... ok, I've created #110 with some tweaks to the a11y rules.